tutorials/intermediate/arranging_axes.py

```diff
@@ -0,0 +1,350 @@
+"""
+===================================
+Arranging multiple Axes in a Figure
+===================================
+
+Often more than one axes is wanted on a figure at a time, and usually
```

Comment:
```suggestion
Often more than one Axes is wanted on a figure at a time, and usually
```

Change to uppercase everywhere per #18726?